### DISCUSSION TITLE
Version 1.0

### DISCUSSION THREAD

**Discussion by writing-shed** [rajdeep/proton#122] [General]
Hi Rajdeep, when do you think version 1 will be released?

**Comment by rajdeep** [comment]
Hi Keith,
Proton 1.0 is not planned in short term yet. Before moving to 1.0, I would like to make some performance optimizations (which have been really tricky because of TextKit architecture) and also possibly add API that is more geared towards SwiftUI.

Regards,
Rajdeep

**Reply by easiwriter** [reply]
Hi
I've been tied up with other things like SwiftData and The Composable Architecture, but am now back with Proton. Are you still developing it?

**Reply by rajdeep** [reply]
@writing-shed, Proton is still under active development though I've not made any releases lately. Given the backlog of items I've and things that I still want to do, it's still tracking well towards 1.0.
